scss/components/_alert.scss

```diff
@@ -222,15 +222,15 @@
     &--light {
         padding:0 0 0 $baseline * 2;
         background:none;
-        border-left: 8px $poppy solid;
-        color:$thunder;
+        border-left: 8px $red solid;
```

Comment:
$ruby-red is the latest name for this (may also take it away from any reserved colour names?)